blockless-cli/src/cli_clap.rs

```diff
@@ -1,68 +1,117 @@
 #![allow(unused)]
-use blockless::BlocklessConfig;
-use clap::{Arg, ArgMatches, Command};
-
-pub(crate) fn cli_command() -> Command {
-    Command::new("blockless_cli")
-        .arg(
-            Arg::new("input")
-                .help("the input file is wasm file, configure file, or car file")
-                .required(true)
-        )
-        .arg(
-            Arg::new("debug_info")
-                .long("debug_info")
-                .help("the debug info for the runtime.")
-                .required(false),
-        )
-        .arg(
-            Arg::new("fs_root_path")
-                .long("fs_root_path")
-                .help("the root directory for the runtime.")
-                .required(false),
-        )
-        .arg(
-            Arg::new("runtime_logger")
-                .long("runtime_logger")
-                .help("the logger file for the runtime.")
-                .required(false),
-        )
-        .arg(
-            Arg::new("limited_memory")
-                .long("limited_memory")
-                .help("the limited memory for the runtime, default is infine.")
-                .required(false),
-        )
-        .arg(
-            Arg::new("run_time")
-                .long("run_time")
-                .help("the run time for the runtime, default is infine.")
-                .required(false),
-        )
-        .arg(
-            Arg::new("entry")
-                .long("entry")
-                .help("the entry for wasm, default is _start")
-                .required(false),
-        )
-        .arg(
-            Arg::new("limited_fuel")
-                .long("limited_fuel")
-                .help("the limited fuel for runtime, default is infine")
-                .required(false),
-        )
-        .allow_missing_positional(true)
+use std::str::FromStr;
+
+use anyhow::{bail, Result};
+use blockless::{BlocklessConfig, BlocklessModule, Permission};
+use clap::{Arg, ArgMatches, Command, Parser};
+use url::Url;
+
+use crate::config::CliConfig;
+
+const INPUT_HELP: &str  = 
+    "the input file is wasm file, configure file, or car file";
+
+const DEBUG_INFO_HELP: &str  = 
+    "the input file is wasm file, configure file, or car file";
```

Comment:
Should be `the debug info for the runtime` or something like that?

Reply:
yes you are right